sale_order_invoicing_policy/models/sale_order.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    sale_order_invoice_policy = fields.Selection(
```

Comment:
I'd prefer to make the field name just `invoice_policy` to be concise and be more consistent with style (product is `product_id` in all the models).